### PR TITLE
Goobi adapter terraform

### PR DESCRIPTION
**Pull request by mklander**
### What is this PR trying to achieve?
Deploy the Goobi adapter
For #1810

### CODE REVIEW

File: catalogue_pipeline/terraform/outputs.tf [hunk 1]
```diff
@@ -2,26 +2,38 @@ output "ecs_services_cluster_name" {
...
   value = "${module.vhs_sourcedata.table_name}"
 }
 
-output "vhs_table_stream_arn" {
+output "vhs_sourcedata_table_stream_arn" {
   value = "${module.vhs_sourcedata.table_stream_arn}"
 }
 
-output "vhs_bucket_name" {
+output "vhs_sourcedata_bucket_name" {
```
Comment (alexwlchan) [inline catalogue_pipeline/terraform/outputs.tf]: 👍

File: catalogue_pipeline/terraform/outputs.tf [hunk 2]
```diff
@@ -2,26 +2,38 @@ output "ecs_services_cluster_name" {
...
   value = "${module.vhs_sourcedata.bucket_name}"
 }
 
+output "vhs_goobi_full_access_policy" {
```
Comment (alexwlchan) [inline catalogue_pipeline/terraform/outputs.tf]: `vhs_goobi_mets_full_access_policy` for completeness?

File: goobi_adapter/goobi_reader/src/main/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <!-- encoders are assigned the type
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <root level="info">
```
Comment (alexwlchan) [inline goobi_adapter/goobi_reader/src/main/resources/logback.xml]: Embrace the brave new world of #2209?

File: goobi_adapter/terraform/goobi_adapter/iam_policy_document.tf [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    ]
+
+    resources = [
+      "${data.aws_s3_bucket.goobi_data.arn}",
+      "${data.aws_s3_bucket.goobi_data.arn}/*",
+    ]
+  }
+}
+
+data "aws_iam_policy_document" "read_from_goobi_items_q" {
```
Comment (alexwlchan) [inline goobi_adapter/terraform/goobi_adapter/iam_policy_document.tf]: This is provided as an output of our SQS module – try `${module.goobi_items_queue.read_policy}`.

File: goobi_adapter/terraform/goobi_adapter/iam_policy_document.tf [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    resources = [
+      "${module.goobi_items_queue.arn}",
+    ]
+  }
+}
+
+data "aws_iam_policy_document" "allow_cloudwatch_push_metrics" {
+  statement {
+    actions = [
+      "cloudwatch:PutMetricData",
```
Comment (alexwlchan) [inline goobi_adapter/terraform/goobi_adapter/iam_policy_document.tf]: Colour me surprised that this isn't provided by a module!

File: goobi_adapter/terraform/topics.tf [hunk 6]
```diff
@@ -0,0 +1,28 @@
+data "aws_iam_policy_document" "publish_to_topic" {
```
Comment (alexwlchan) [inline goobi_adapter/terraform/topics.tf]: Our SNS module provides a publish policy output.

Comment (alicefuzier) [reply goobi_adapter/terraform/topics.tf]: yeah, but we can't use that for SNS bucket notification cause you need the condition on the bucket ARN in the policy

File: goobi_adapter/goobi_reader/src/main/resources/logback.xml (hunk 3 above)
Comment (alicefuzier) [reply goobi_adapter/goobi_reader/src/main/resources/logback.xml]: Yep, but it's not in master yet and even in that PR is only for two apps. I'm guessing the reason for that is to try it out first so I don't really want to add it here until it's standard

File: goobi_adapter/terraform/goobi_adapter/iam_policy_document.tf [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    actions = [
+      "sqs:DeleteMessage",
+      "sqs:ReceiveMessage",
+    ]
+
+    resources = [
+      "${module.goobi_mets_queue.arn}",
+    ]
+  }
+}
```
Comment (alexwlchan) [inline goobi_adapter/terraform/goobi_adapter/iam_policy_document.tf]: I think I commented already that this should be provided by the module?

**Pull request merged**